examples/private_set_intersection/client/rust/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "private_set_intersection_client"
+version = "0.1.0"
+authors = ["Ivan Petrov <ivanpetrov@google.com>"]
+edition = "2018"
+license = "Apache-2.0"
+
+[lib]
+name = "private_set_intersection_client"
+path = "src/lib.rs"
+
+[[bin]]
+name = "psi_client_bin"
+path = "src/main.rs"
```

Comment:
Why are these both needed?

Reply:
`lib` is needed for integration tests, in order to include `private_set_intersection_client::PrivateSetIntersectionClient`